src/types/ut/Types.Unit.Test.vcxproj

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(SolutionDir)src\common.build.pre.props" />
+  <ItemGroup>
+    <ClCompile Include="UuidTests.cpp" />
+    <ClCompile Include="..\precomp.cpp">
+      <PrecompiledHeader>Create</PrecompiledHeader>
+    </ClCompile>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\lib\types.vcxproj">
+      <Project>{18d09a24-8240-42d6-8cb6-236eee820263}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\precomp.h" />
+  </ItemGroup>
+  <PropertyGroup>
+    <ProjectGuid>{34de34d3-1cd6-4ee3-8bd9-a26b5b27ec73}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>TypesUnitTests</RootNamespace>
+    <ProjectName>Types.Unit.Test</ProjectName>
+    <TargetName>Types.Unit.Test</TargetName>

```

Comment:
Probably another area where cleanup/reconciliation is needed.

The Host Unit Tests (the O.G. of our testing libraries) says

```
<ProjectName>Host.Tests.Unit</ProjectName>
<TargetName>Conhost.Unit.Tests</TargetName>
```

So at least the target name might need to be pluralized to match the pattern in the CI (i.e. `Types.Unit.Tests`).
I think ProjectName only affects the sort order inside Visual Studio so similar projects are sorted together visually.

I'd recommend aligning at least the `TargetName` (or confirming the CI pattern isn't looking for the plural) here and then filing a follow on task to make the test libraries named more consistently.